templates/discussions-dashboard.php

```diff
@@ -0,0 +1,210 @@
+			$original_id = Helper_Translation_Discussion::get_original_from_post_id( $_post_id );
+			if ( ! $original_id ) {
+				continue;
+			}
+
+			$original             = GP::$original->get( $original_id );
+			$first_comment        = reset( $post_comments );
+			$no_of_other_comments = count( $post_comments ) - 1;
+			?>
+			<tr>
+				<td>
+					<?php
+					echo esc_html( $original->singular );
+					if ( isset( $bulk_comments[ $original_id ] ) ) {
+						?>
+						<details>
+							<summary class="other-comments">
+							<?php
+							/* translators: number of other originals in the bulk rejection. */
+							printf( '+ ' . _n( '%s Other', '%s Others', count( $bulk_comments[ $original_id ] ) ), number_format_i18n( count( $bulk_comments[ $original_id ] ) ) );
+							?>
+						</summary>
+						<ul>
+						<?php
+						foreach ( $bulk_comments[ $original_id ] as $_comment ) {
+							?>
+							<li><a href="<?php echo esc_attr( $_comment->comment_content ); ?>"><?php echo esc_html( $_comment->comment_content ); ?></a></li>
+							<?php
+						}
+						?>
+						</ul>
+						</details>
+						<?php
+					}
+					?>
+				</td>
+				 <td>
+					 <?php if ( ! $first_comment->comment_content ) : ?>
+						
+							<?php
+							$comment_reason  = get_comment_meta( $first_comment->comment_ID, 'reject_reason', true );
+							$number_of_items = count( $comment_reason );
+							$counter         = 0;
+							$comment_reasons = Helper_Translation_Discussion::get_comment_reasons();
+							foreach ( $comment_reason as $reason ) {
+								echo wp_kses(
+									sprintf(
+									/* translators: 1: The comment reason */
+										__( '<span>%1$s</span>', 'glotpress' ),
+										$comment_reasons[ $reason ]['name'],
+									),
+									array(
+										'span' => array(),
+									)
+								);
+
+								if ( ++$counter < $number_of_items ) {
+									echo ', ';
+								}
+							}
+							?>
+						
+					<?php else : ?>
+						<a href="<?php echo esc_url( get_comment_link( $first_comment ) ); ?>"><?php echo esc_html( get_comment_excerpt( $first_comment ) ); ?></a>
+						<?php if ( $no_of_other_comments > 0 ) : ?>
+							<br>
+							<?php /* translators: number of comments. */ ?>
+							<a class="other-comments" href="<?php echo esc_url( get_comment_link( $first_comment ) ); ?>"> + <?php printf( _n( '%s Comment', '%s Comments', $no_of_other_comments ), number_format_i18n( $no_of_other_comments ) ); ?></a>
+						<?php endif; ?>
+					<?php endif; ?>
+				</td>
+				<td><?php echo esc_html( $first_comment->comment_author ); ?></td>
+				<td><?php echo esc_html( $first_comment->comment_date ); ?></td>
+			</tr>
+			<?php
+		}
+		?>
+		
+	</tbody>
+	
+</table>
+<?php
+	$current_page = max( 1, get_query_var( 'page' ) );
+	echo wp_kses(
+		paginate_links(
+			array(
+				'base'      => add_query_arg( 'page', '%#%' ),
+				'format'    => '?page=%#%',
+				'current'   => $current_page,
+				'total'     => count( $comments ),
```

Comment:
```suggestion
				'total'     => $comments_query->found_comments / $page_size,
```

Reply:
http://developer.wordpress.org/reference/classes/wp_comment_query/
```
         * Filtering functions that require pagination information are encouraged to set
         * the `found_comments` and `max_num_pages` properties of the WP_Comment_Query object,
         * passed to the filter by reference. If WP_Comment_Query does not perform a database
         * query, it will not have enough information to generate these values itself.
         *
```

Reply:
This returned `0`. Eventually had to count all the comments and divide by `comments_per_page`